client/dataEntry.js

```diff
@@ -139,6 +139,24 @@ function getSolveWarnings(resultId, solveIndex) {
   return warnings;
 }
 
+function updateWarningContents () {
+  let data = Template.currentData();
+  let selectedResultId = selectedResultIdReact.get();
+  let warnings = getSolveWarnings(selectedResultId, data.index);
+  let $warningIcon = template.$('.solve-time-warning-icon');
```

Comment:
Maybe we should call `updateWarningContents` right after saving to the database, since then the data will be in the database and we can correctly compute the warnings?